openshift/template.yaml

```diff
@@ -286,6 +287,58 @@ objects:
                     key: ${JOBS_AUTO_REPORT_SLACK_CHANNEL_ID_KEY}
                     name: ${JOBS_AUTO_REPORT_SLACK_CHANNEL_ID_NAME}
 
+- apiVersion: batch/v1
+  kind: CronJob
+  metadata:
+    name: elasticsearch-cleanup
+  spec:
+    concurrencyPolicy: ${ELASTICSEARCH_CLEANUP_CONCURRENCY_POLICY}
+    schedule: ${ELASTICSEARCH_CLEANUP_SCHEDULE}
+    suspend: ${{ELASTICSEARCH_CLEANUP_SUSPEND}}
+    jobTemplate:
+      spec:
+        ttlSecondsAfterFinished: 10800 # keep last 3 runs (3 * 1h)
+        backoffLimit: 3
+        template:
+          spec:
+            restartPolicy: OnFailure
+            serviceAccountName: assisted-service
+            containers:
+            - name: elasticsearch-cleanup
+              image: ${IMAGE_NAME}:${IMAGE_TAG}
+              imagePullPolicy: ${IMAGE_PULL_POLICY}
+              command: 
+              - elasticsearch-cleanup
+              resources:
```

Comment:
Resource policies also can be parametrized, so that we can adjust to real consumption more easily